sensor/common/sensor/sensor.go

```diff
@@ -305,12 +306,26 @@ func (s *Sensor) Stop() {
 		c.Stop(nil)
 	}
 
+	log.Infof("Sensor stop was called. Stopping all listeners")
+
 	if s.profilingServer != nil {
 		if err := s.profilingServer.Close(); err != nil {
 			log.Errorf("Error closing profiling server: %v", err)
 		}
 	}
 
+	if s.server != nil {
+		if !s.server.Stop() {
+			log.Warnf("Sensor gRPC server stop was called more than once")
+		}
+	}
```

Comment:
Why not like this?
```suggestion
	if s.server != nil && !s.server.Stop() {
		log.Warnf("Sensor gRPC server stop was called more than once")
	}
```

Reply:
Because I only want to show the warning if the result of `.Stop()` is false. With your suggestion sensor will show the warning ` Sensor gRPC server stop was called more than once` if `s.server` is `nil`, which is not correct.

Reply:
I propose a solution with `&&`, not with `||`. This is only a syntactical difference, behavior should stay the same.

Reply:
I don't know what I was thinking. You're right, updated the code.